### PR TITLE
Fixes #7716: Restart foreman-tasks when katello.yml changes.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by jlsherrill** [comment]
Should this also notify the httpd service?

**Comment by ehelms** [comment]
Good call. I was just blindly following the bug.

**Comment by ehelms** [comment]
@jlsherrill Added httpd

File: manifests/config.pp [hunk 1]
```diff
@@ -15,6 +15,7 @@
     group   => $katello::group,
     mode    => '0644',
     before  => [Class['foreman::database'], Exec['foreman-rake-db:migrate']],
+    notify  => [Service['foreman-tasks'], Service['httpd']],
```
Comment (ekohl) [inline manifests/config.pp]: I thought this was the same as https://github.com/Katello/puppet-katello/pull/62, but also here you may want to notify ```Class['foreman::service']```.

**Comment by ehelms** [comment]
Green here to!

**Comment by iNecas** [comment]
ACK

**Pull request merged**